### ISSUE TITLE
Add category function for installed Extensions 

### ISSUE THREAD

**Issue by Zeal-L** [microsoft/vscode#174685]
Given the large number of Extensions (100+) that users may install. Some of these Extensions may be serving the c++ workspace and some the python workspace.

I currently have to check one by one each time I create a new workspace and turn off the Extensions I don't need. If I don't turn them off, it's too slow to start vscode.

It would be nice if there was a feature to group installed Extensions . This would allow users to batch start the Extensions used for the corresponding language when creating a new workspace.

**Comment by gjsjohnmurray** [comment]
Have you looked at the recently-added [Profiles](https://code.visualstudio.com/updates/v1_75#_profiles) feature?

**Comment by Zeal-L** [comment]
> Have you looked at the recently-added [Profiles](https://code.visualstudio.com/updates/v1_75#_profiles) feature?

Yes. I've looked at it and I think the feature essentially still requires the user to check one by one for Extensions that need to be turned on.

For example, I now have a workspace with python Extensions and rust Extensions that I need to use. I can't use this feature to import two Profiles at the same time. 

I'd like to be able to have user-defined Extensions groups and be able to quickly turn on and off a group of Extensions with one click.

**Comment by sandy081** [comment]
/duplicate

https://github.com/microsoft/vscode/issues/20599
